src/scripts/OSFramework/OSUI/GlobalEnum.ts

```diff
@@ -16,13 +16,15 @@ namespace OSFramework.OSUI.GlobalEnum {
 		AsideExpandable = 'aside-expandable',
 		Container = 'screen-container',
 		Content = 'content',
+		DarkMode = 'os-dark-mode',
 		DeprecatedSubmenu = 'submenu',
 		Footer = 'footer',
 		Header = 'header',
 		HeaderHideOnScroll = 'hide-header-on-scroll',
 		HeaderIsFixed = 'fixed-header',
 		HeaderIsVisible = 'header-is--visible',
 		HeaderTopContent = 'header-top-content',
+		HighContrast = 'os-hight-contrast',
```

Comment:
This will need to be changed in all files
```suggestion
		HighContrast = 'os-high-contrast',
```

Reply:
Done!